index.ar.md

```diff
@@ -0,0 +1,2856 @@
+
+_2021-03-23_
+
+<p dir='rtl' align='right'>
+ريتشارد ماثيو ستالمان (Richard M. Stallman) والمعروف ب RMS، كان ولزال لوقت طويل قوة خطرة في مجتمع البرمجيات الحرة. إنه يظهر نفسه على أنه كارها للمرأة (misogynist)، شخصا ضد المعاقين (ableist)، وكارها للمتحولين جنسيا (transphobic)، وإتهامات أخرى عديدة. لا مكان لهذه الأنواع من المعتقدات (التوجهات) في البرمجيات الحرة، الحقوق الرقمية، ومجتمعات التكنولوجيا والتقنية. مع إعادته مؤخرا إلى مجلس إدارة مؤسسة البرمجيات الحرة، نحن  ندعوا مجلس إدارة FSF إلى التنحي بإكمله وإزاحة RMS من جميع المناصب. 
+</p>
+  
+<p dir='rtl' align='right'>
+  نحن الموقعون أدناه، نؤمن بضرورة الاستقلالية الرقمية، والدور القوي الذي تلعبه حرية المستخدم في حماية حقوق الإنسان الأساسية. من أجل تحقيق الوعد بأن كل شيء تجعله حرية البرمجيات ممكنا، يجب أن يكون هناك تغيير جذري داخل المجتمع. نحن نؤمن بالحاضر والمستقبل، حيث تعمل جميع التقنيات على تمكين الناس وليس اضطهادهم. نحن نعلم بأن هذا ممكن فقط في عالم تبنى فيه التكنولوجيا لاحترام حقوقنا في أكثر مستوياتها التأسيسية. في حين تم الترويج لهذه الافكار  (وأخرى) بشكل ما من قبل ريتشارد ماثيو ستالمان (Richard M. Stallman)، إلا أنه لا يتحدث نيابة عنا. نحن لا نتغاضى عن أفعاله وآرائه. نحن لا نعترف بقيادته أو قيادة مؤسسة البرمجيات الحرة كما هي اليوم.
+</p>
+
+<p dir='rtl' align='right'>
+كان هناك تسامح كافي مع أفكار وسلوكيات RMS البغيضة. لا يمكننا الاستمرار في ترك شخص واحد يفسد معنى عملنا. ليس لمجتمعاتنا مساحة لأشخاص مثل ريتشارد ماثيو ستالمان (Richard M. Stallman)، ولن نستمر في المعاناة من سلوكياته، أو منحه دورا قياديا، أو قبوله هو وايدولوجيته المؤذية والخطيرة. 
+</p>
+
+<p dir='rtl' align='right'>
+  نحن ندعوا لإزالة مجلس إدارة مؤسسة البرمجيات الحرة بالكامل، هؤلاء هم الأشخاص الذين قاموا بتمكين RMS لسنوات، يبرهنون على ذلك مرة أخرى من خلال السماح له بالعودة لمجلس إدارة FSF. حان الوقت ل RMS للتراجع عن البرمجيات الحرة، أخلاقيات التكنولوجيا، الحقوق الرقمية، ومجتمعات التكنولوجيا. لأنه لا يستطيع توفير القيادة التي نحتاجها. نحن ندعوا أيضا لعزل ريتشارد ماثيو ستالمان (Richard M. Stallman) من مجميع المنصاب القيادية، بما في ذلك مشروع جنو. 
+</p>
+
+<p dir='rtl' align='right'>
+  نحث أيضا أولائك الذين هم في وضع يسمح لهم، بالتوقف عن دعم مؤسسة البرمجيات الحرة. رفض المساهمة في المشاريع التي ترجع إلى FSF و RMS. التوقف عن التحدث وحضور أحداث FSF ، أو الأحداث التي تسمح بإستقبال والترحيب ب RMS وعلامته التجارية المتعصبة. نحن نطلب من المساهمين في مشاريع البرمجيات الحرة في إتخاذ موقف ضد التعصب والكراهية في مشاريعهم، وأثناء القيام بذلك أخبر المجتمعات ومؤسسة FSF لماذا. 
+</p>
+  
+<p dir='rtl' align='right'>
+    لقد قمنا بتفصيل العديد من الحوادث العامة لسلوكيات RMS، البعض منا لديه قصصه الخاصة حول RMS وتفاعلاتنا معه
+    ، أشياء لم يتم إلتقاطها في ردود الإيمايلات أو في الفيديو، نأمل أن تقرأ ما تمت مشاركته وأن تنظر في الضرر الذي ألحقه في بمجتمعنا والآخرين.
+    التفاصيل: https://rms-open-letter.github.io/appendix
+</p>
+  
+
+<p dir='rtl' align='right'>
+    للتوقيع يرجى القيام/إرسال digitalautonomy إلى  riseup.net  أو القيام ب 
+  pull request: https://github.com/rms-open-letter/rms-open-letter.github.io/pulls
+</p>
+
```

Comment:
Please remove everything from here down as we are not including signatures on the translations.